condor/condor_meter

```diff
@@ -711,7 +710,19 @@ def classadToJUR(classad):
 
     setIfExists(r.Queue, classad, "JobUniverse", "Condor's JobUniverse field", setstr=True)
     setIfExists(r.NodeCount, classad, 'MaxHosts', "max")
-
+    ########################################################################################
+    ########################################################################################
+    # Code added to send to arbitrary Ads SOFTWARE-2714
+    ArbitraryJobAds = GratiaCore.Config.getConfigAttribute("ArbitraryJobAdAttributes")
+    DebugPrint(5, "ArbitraryJobAd: %s" % str(ArbitraryJobAds))
+    Arbirtray_adlist = ArbitraryJobAds.split(",")
```

Comment:
They are not equivalent regular expressions, but I guess that they behave similarly in this particular case due to a oddity (IMO) of Python. The docs for re.split() say, “Note that split will never split a string on an empty pattern match”, which is key here.